awx/main/models/inventory.py

```diff
@@ -872,6 +872,13 @@ class Meta:
         default='',
         help_text=_('Inventory source variables in YAML or JSON format.'),
     )
+    scm_branch = models.CharField(
+        max_length=1024,
+        default='',
+        blank=True,
+        null=True,
+        help_text=_('Branch to use for inventory source. Project default used if blank. Only allowed if project allow_override field is set to true.'),
```

Comment:
very nit-picky, but I would say "inventory update" as opposed to "inventory source", because this field applies to both models, and I think that wording would make sense in both cases. Or just say "Inventory source SCM branch"

Reply:
changed the text, thanks